oci/private/image.bzl

```diff
@@ -85,11 +85,6 @@ If `group/gid` is not specified, the default group and supplementary groups of t
     "_empty_tar": attr.label(default = "empty.tar", allow_single_file = True),
 }
 
-def _format_string_to_string_tuple(kv):
```

Comment:
this is unused. it's removed in a couple other PRs which are blocked. i want to remove this now.